src/virtual-repeat.js

```diff
@@ -411,7 +411,14 @@ export class VirtualRepeat extends AbstractRepeater {
     let firstViewElement = this.view(0).lastChild;
     this.itemHeight = calcOuterHeight(firstViewElement);
     if (this.itemHeight <= 0) {
-      throw new Error('Could not calculate item height');
+      this._sizeInterval = setInterval(()=>{
```

Comment:
Ohh, good call. Do you think 500ms is fine for the interval? I just made it match the interval (time wise) with the one defined earlier in regards to scrolling. Possibly make it faster, slower?